sdk-extension/opentelemetry-sdk-extension-aws/src/opentelemetry/sdk/extension/aws/resource/eks.py

```diff
@@ -74,11 +74,7 @@ def _get_cluster_info(cred_value):
     )
 
 
-def _get_cluster_name():
-    cred_value = _get_k8s_cred_value()
-    if not _is_eks(cred_value):
-        return Resource.get_empty()
-
+def _get_cluster_name(cred_value):
```

Comment:
maybe for clarity (too bad the type checker doesn't actually run on here)

```suggestion
def _get_cluster_name(cred_value) -> str:
```

Reply:
Great idea! Thanks! 🙂